### PR TITLE
Add VyOS support

### PR DESCRIPTION
**Pull request by Pouriya-Inmanta**
# Summary

Adds support and instruction for VyOS containers.

### CODE REVIEW

File: Networking/Vyos/project.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Cisco Examples
```
Comment (bartv) [inline Networking/Vyos/project.yml:1]: copy paste error

Comment (Pouriya-Inmanta) [reply Networking/Vyos/project.yml:1]: yes, still a lot to fix.

File: Networking/Vyos/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# VyOS Module
+
+The provided examples in the `*.cf` files can be applied on `VyOS` devices either on:
+
+* A router that you already have and by changing the IP address, port, etc... in the examples files
+* Or using [containerlab](https://containerlab.srlinux.dev/) with the provided [topology file](containerlab/topology.yml) and the `startup config` files for [vyos1](containerlab/vyos1.cfg) and [vyos2](containerlab/vyos2.cfg).
+
+In case of choosing the `containerlab` option, it is worthwhile to mention that you need to have a `VyOS` Docker image present on the host machine running `containerlab` and having it **tagged** similar to the topology file; in this case: `vyos/vyos:1.2.8`.
+
+Inmanta [Service Orchestrator](https://inmanta.com/service-orchestrator/) is another containerized piece that can supply a GUI, providing many useful information such as deployment process, agents status and so much more. This container is not a necessity to try out these examples but it is nice to have. We can provide access to it if you contact us.
```
Comment (wouterdb) [inline Networking/Vyos/README.md:10]: @bartv don't we provide the open source orchestrator publicly?

File: Networking/Vyos/on_openstack.cf [hunk 3]
```diff
@@ -0,0 +1,123 @@
+import openstack
```
Comment (wouterdb) [inline Networking/Vyos/on_openstack.cf:1]: I think this one can be omitted

File: Networking/Vyos/README.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+The programs included with the Debian GNU/Linux system are free software;
+the exact distribution terms for each program are described in the
+individual files in /usr/share/doc/*/copyright.
+
+Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
+permitted by applicable law.
+```
+
+## Additional Containerlab Commands
```
Comment (wouterdb) [inline Networking/Vyos/README.md:89]: Perhaps also add the command on how to deploy one of the examples to the clab?

Comment (Pouriya-Inmanta) [reply Networking/Vyos/README.md]: Done

File: Networking/Vyos/README.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+Delete the deployed LAB:
+
+```sh
+sudo clab destroy -t topology.yml
+```
+
+Export the model to Inmanta server:
+
+```sh
+inmanta -vvv export -f ospf.cf -e <environment_id> --server_address <server_ip_address>
```
Comment (wouterdb) [inline Networking/Vyos/README.md:112]: I would also add `inmanta-cli --host <orchestrator_ip>  project create -n test` and `inmanta-cli --host <orchestrator_ip>  environment create -p test -n vyos --save` 
Then the export doesn't need the last two arguments, but I leave that up to you.

(best to try the commands, I tend to make typos)

**Review by wouterdb — APPROVED** [review]

File: Networking/Vyos/README.md (hunk 5 above)
Comment (Pouriya-Inmanta) [reply Networking/Vyos/README.md:112]: I will try the commands and add them as well.

Comment (Pouriya-Inmanta) [reply Networking/Vyos/README.md:112]: Verified and added the `inmanta-cli` commands.

**Pull request merged**